lib/console/deployments/pr/impl/gitlab.ex

```diff
@@ -92,18 +130,40 @@ defmodule Console.Deployments.Pr.Impl.Gitlab do
   defp owner(_), do: nil
 
   defp connection(conn) do
-    with {:ok, url, token} <- url_and_token(conn, "https://gitlab.com"),
+    with {:ok, url, token} <- url_and_token(conn, @gitlab_api_url),
       do: Connection.new(url, token)
   end
 
+  defp uri_encode(str), do: URI.encode(str, & &1 != ?/ and URI.char_unescaped?(&1))
+
   defp get_pull_id(url) do
-    with {:ok, %URI{path: "/" <> path}} <- URI.parse(url),
-         [group, "/merge_requests/" <> number] <- String.split(path, "-") do
-      {:ok, group, number}
+    with %URI{path: "/" <> path} <- URI.parse(url),
```

Comment:
this is going to cause a regression on the `review` function at least, maybe a few others